Open trailers in tv web interface & others

persi-persu opened the pull request.

Commit:
Open trailers in tv web interface

- Add tv web interface support in webview

courville commented:
Unrelated but shared new test build same link with jcifsng that is tested to work.

Commit:
Fix non scraped video icon in list

- Fix episode poster in season

courville commented on src/leanback/java/com/archos/mediacenter/video/leanback/details/VideoDetailsFragment.java:
```diff
@@ -349,8 +350,16 @@ public void onItemClicked(Presenter.ViewHolder itemViewHolder, Object item, RowP
+                    else {
+                        String url = ((ScraperTrailer)item).getUrl().toString().replace("https://www.youtube.com/watch", "https://www.youtube.com/tv#/watch");
+
+                        WebUtils.openWebLink(getActivity(), url);
```
I had users complaining not using Android Youtube app to watch trailers hence 5d46e6c15b24bc617aa694aa1fca3ffb77614f68
But I like your tv#/watch trick which is enough to convince me.

courville commented on src/leanback/java/com/archos/mediacenter/video/leanback/details/VideoDetailsFragment.java:
```diff
@@ -349,8 +350,16 @@ public void onItemClicked(Presenter.ViewHolder itemViewHolder, Object item, RowP
-                    //WebUtils.openWebLink(getActivity(), ((ScraperTrailer)item).getUrl().toString());
+                    ActivityInfo activityInfo = browserIntent.resolveActivityInfo(getActivity().getPackageManager(), browserIntent.getFlags());
+                    
+                    if (activityInfo != null && !activityInfo.processName.equals("com.google.android.tv.frameworkpackagestubs")) {
```
What is the use of checking com.google.android.tv.frameworkpackagestubs ?

Commit:
Change padding of info icons

- Minor text change in sources
- Add audio badge in sources
- Fix all episodes watched when multiple sources

Commit:
Fix source of select video in search

Commit:
Fix keyboard submit in search

Commit:
Improve tv web interface support

Commit:
Update background color with poster

Commit:
Update backdrop in tv show

- Minor text changes

Commit:
Fix tv menu item focus

Commit:
Update and use subtitles wizard

Commit:
Show title with player controls

Commit:
Add reset last played row setting

Commit:
Update background color after unindex/unscrap

courville reviewed (commented):
I very much like reviving the subtitle wizard (even if it is not so leanback looking compliant).
For now it works for SMB shares and local storage but not for SFTP or UPnP shares.
Note that there is no error in this case but it just says no srt files found.
Note that there is no standard way to get subs files other than bruteforce for UPnP (standard lacks this).
It should be fixable for sftp though.
Perhaps we need either to fix it or disable the button in that case.

Commit:
Show subtitles wizard if supported
